tensorflow/workspace2.bzl

```
@@ -201,7 +201,7 @@ def _tf_repositories():
     tf_http_archive(
         name = "mkl_dnn_acl_compatible",
         build_file = "//third_party/mkl_dnn:mkldnn_acl.BUILD",
-        patch_file = ["//third_party/mkl_dnn:onednn_acl_threadcap.patch", "//third_party/mkl_dnn:onednn_acl_fixed_format_kernels.patch", "//third_party/mkl_dnn:onednn_acl_depthwise_convolution.patch"],
+	patch_file = ["//third_party/mkl_dnn:onednn_acl_threadcap.patch", "//third_party/mkl_dnn:onednn_acl_fixed_format_kernels.patch", "//third_party/mkl_dnn:onednn_acl_depthwise_convolution.patch", "//third_party/mkl_dnn:onednn_reorder_padded.patch", "//third_party/mkl_dnn:onednn_reorder_update.patch","//third_party/mkl_dnn:onednn_acl_threadpoolscheduler.patch"],
```

Nit: The list is getting long, let's make it multi-line.
```suggestion
	patch_file = [
		"//third_party/mkl_dnn:onednn_acl_threadcap.patch", 
		"//third_party/mkl_dnn:onednn_acl_fixed_format_kernels.patch", 
		"//third_party/mkl_dnn:onednn_acl_depthwise_convolution.patch", 
		"//third_party/mkl_dnn:onednn_reorder_padded.patch", 
	    "//third_party/mkl_dnn:onednn_reorder_update.patch","//third_party/mkl_dnn:onednn_acl_threadpoolscheduler.patch"
	],
```